middleware/rate_limiter_v2_api.rb

```diff
@@ -0,0 +1,40 @@
+require 'mixins/client_ip'
+require 'mixins/user_reset_interval'
+require 'base_rate_limiter'
+
+module CloudFoundry
+  module Middleware
+    REQUEST_COUNTER_V2_API = BaseRequestCounter.new
```

Comment:
This constant can be moved into the `RateLimiterV2API` class and renamed to `REQUEST_COUNTER`.